analyzers/tests/SonarAnalyzer.UnitTest/Metrics/CSharpExecutableLinesMetricTest.cs

```diff
@@ -30,885 +29,874 @@ public class CSharpExecutableLinesMetricTest
         [TestMethod]
         public void No_Executable_Lines() =>
             AssertLinesOfCode(
-              @"using System;
-                using System.Linq;
+@"using System;
```

Comment:
There's a lot of whitespace cleanup for readability. This file should be easier to review per commit